src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/Internal/Linux/LinuxUtilizationProvider.cs

```diff
@@ -56,8 +51,8 @@ internal sealed class LinuxUtilizationProvider : ISnapshotProvider
         _scale = hostCpus * Hundred / availableCpus;
         _scaleForTrackerApi = hostCpus / availableCpus;
 
-        _ = meter.CreateObservableGauge<double>(name: LinuxResourceUtilizationCounters.CpuConsumptionPercentage, observeValue: CpuPercentage);
-        _ = meter.CreateObservableGauge<double>(name: LinuxResourceUtilizationCounters.MemoryConsumptionPercentage, observeValue: MemoryPercentage);
+        _ = meter.CreateObservableGauge<double>(name: ResourceUtilizationCounters.CpuConsumptionPercentage, observeValue: CpuPercentage);
```

Comment:
I would have rename it to 'ResourceUtilizationMetrics'. Then we could have counters, gauges and up/down metric names in one class (better discoverability).